src/cli/generator/generator.rs

```diff
@@ -89,6 +84,16 @@ impl Generator {
             vars: &Default::default(),
             headers: Default::default(),
         };
+
+        if let Ok(mustache) = Mustache::parse(&config_content) {
+            config_content = mustache.render(&reader_context);
+        }
+
+        let config: Config = match source {
+            ConfigSource::Json => serde_json::from_str(&config_content)?,
+            ConfigSource::Yml => serde_yaml::from_str(&config_content)?,
+        };
+
```

Comment:
This is an interesting approach @ssddOnTop. I think in this case we don't need a lot of logic that we have implemented right now to resolve headers and secret. Please drop that logic.